### DISCUSSION TITLE
Importing from WooCommerce

### DISCUSSION THREAD

**Discussion by melliston** [medusajs/medusa#8864] [Q&A]
Hey loving the project and looking to migrate an existing store from WooCommerce to Medusa.

I was trying to see if there is a way to migrates:

Categories
Products
Users
Orders
User Order History
Reviews 

I was looking at doing it direct in the DB. Exporting from one and then transposing to Medusa directly in the DB. However looking into it I dont think this will be possible or as easy as I thought it would be. The keys look like they are system generate.

Could anyone advise the best strategy to migrate. I have been looking at the shopify method and that could be a good starter potentially if I just convert it woo commerce. 

There are about 8000 products and customers of over 8 years so cant really start fresh.

Any advice would be greatly appreciated.

**Comment by 37scripts** [comment]
found this : https://weframetech.com/services/woo-commerce-to-medusa-js
